api/v1alpha1/dspipeline_types.go

```diff
@@ -178,6 +206,7 @@ type Minio struct {
 }
 
 type MLMD struct {
+	// Default: false
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
 	Deploy  bool `json:"deploy"`
```

Comment:
"Enable DS Pipelines Operator management of MLMD. Setting Deploy to false disables operator reconciliation."